spec/controllers/users_controller_spec.rb

```diff
@@ -68,4 +69,24 @@
       end
     end
   end
+
+  describe "#destroy" do
+    let(:deletable) { FactoryGirl.create :alice }
+    before(:each) do
+      sign_in admin
+    end
+    describe "destroying users" do
+      it "should not delete user with GenericFiles" do
```

Comment:
I need some help on this one: this spec fails (it deletes the user even though the user has a deposit, which should block it at https://github.com/ualbertalib/HydraNorth/pull/1001/files#diff-4e05ad0d64e6100656b63ad1e78f32c5R27 ). Why doesn't the controller detect the deposit?